app/manifest/_base.json

```diff
@@ -59,7 +59,7 @@
     "512": "images/icon-512.png"
   },
   "manifest_version": 2,
-  "name": "__MSG_appName__",
+  "name": "Amino",
```

Comment:
```suggestion
  "name": "__MSG_appName__",
```

Reply:
fixed in 66ce7f463bb930ba2978b71767c297777aaa69b9

Reply:
- Verified